package.json

```diff
@@ -1,10 +1,57 @@
 {
+  "license": "Apache-2.0",
   "private": true,
   "workspaces": [
     "@shared/api",
     "@shared/constants",
     "docs",
     "extension",
     "@stellar/lyra-api"
-  ]
+  ],
+  "scripts": {
+    "build": "run-p --print-label build:lyra-api build:docs build:extension",
+    "build:netlify": "run-s build:lyra-api build:docs",
+    "build:lyra-api": "yarn workspace @stellar/lyra-api build",
+    "build:docs": "yarn workspace docs build",
+    "build:extension": "yarn workspace extension build",
+    "start": "run-p --print-label start:lyra-api start:docs start:extension start:unpacked",
+    "start:lyra-api": "yarn workspace @stellar/lyra-api start",
+    "start:docs": "yarn workspace docs start",
+    "start:extension": "yarn workspace extension start",
+    "start:unpacked": "yarn workspace extension start:unpacked-extension",
+    "install-if-package-changed": "git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD | grep --quiet yarn.lock && yarn install || exit 0",
+    "test:ci": "run-p test:ci:lyra-api test:ci:extension",
+    "test:ci:lyra-api": "yarn workspace @stellar/lyra-api test -ci",
+    "test:ci:extension": "yarn workspace extension test -ci",
+    "test": "run-p --print-label test:lyra-api test:extension",
+    "test:lyra-api": "yarn workspace @stellar/lyra-api test -o --watch",
+    "test:extension": "yarn workspace extension test -o --watch"
```

Comment:
I set it up so that `yarn test` runs a watcher that only runs tests for files that have changed since the last commit, and `yarn test:ci` runs all tests assuming a CI environment